examples/websocket_echo_server/lock.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
can we not commit the lock.json? that tends to lead to wtf with tests etc.

is this required? i assume it isn't. (if we do remove, lets add it to the .gitignore)

Reply:
This is in the example. We can totally keep it out. My thought was that it is usually good practice to include it to improve reproducability of the build of this corral project, as the lock is where the version/git-tag pinning happens.